packages/mobile/src/variables/BackgroudnColor.ts

```diff
@@ -0,0 +1,5 @@
+export enum BackgroundColor {
+    BACKGROUNDCOLOR = '#eef0f8',
```

Comment:
wole jako to jest nazwa koloru typu niebieski tak jak na dole jest grey. Jak grey zmieni sie na red to vsc ma narzędzia by podmienic nazwe w całym projekcie